requirements/pip.txt

```diff
@@ -10,5 +10,5 @@ wheel==0.37.1
 # The following packages are considered to be unsafe in a requirements file:
 pip==21.3.1
     # via -r requirements/pip.in
-setuptools==60.1.0
+setuptools==60.2.0
```

Comment:
We had merged the upgrade PR that upgraded the setuptools to version 60 before the issue with it came into out notice. Shouldn't we constraint it in this PR....?